# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/build/moz.build
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/build/moz.build
--- a/media/mtransport/build/moz.build	Tue Sep 17 18:15:23 2019 +0000
+++ b/media/mtransport/build/moz.build	Tue Sep 17 18:15:41 2019 +0000
@@ -38,7 +38,7 @@
 # These files cannot be built in unified mode because of the redefinition of
 # getLogModule, UNIMPLEMENTED, nr_socket_long_term_violation_time,
 # nr_socket_short_term_violation_time, nrappkit/IPDL typedef conflicts in
-# PBrowserOrId and WebrtcProxyChannelChild.
+# PBrowserOrId and WebrtcTCPSocketChild.
 SOURCES += mtransport_cppsrcs
 
 FINAL_LIBRARY = 'xul'